documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -376,33 +265,31 @@ Model In Image model files, application archives, and the WDT installation files
 
        {{%expand "Click here to view the Dockerfile." %}}
        ```
-       # Copyright (c) 2021, Oracle and/or its affiliates.
+       # Copyright (c) 2021, 2022, Oracle and/or its affiliates.
        # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-    
+       
        # This is a sample Dockerfile for supplying Model in Image model files
-       # and a WDT installation in a small separate auxiliary
+       # and a WDT install in a small separate auxiliary image
```

Comment:
I would suggest changing this back to `...WDT installation...`

Reply:
I agree.

Reply:
Fixed this and the previous review comment.